libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx

```diff
@@ -54,13 +51,12 @@ export const getNetworkPanels = (
     );
   });
 
-export const NotDaoNetwork = ({ isSm }: { isSm: boolean }) => {
+export const NotchainId = ({ isSm }: { isSm: boolean }) => {
```

Comment:
I prefer NotDaoNetwork, tbh.

Reply:
whoops that was a find and replace i forgot to change back